### ISSUE TITLE
I have some question about the specific triangulation method used in this paper? Is it Delaunay Triangulation in 2D?

### ISSUE THREAD

**Issue by lingkang** [GhiXu/ACMP#1]

**Comment by GhiXu** [comment]
Yes, ACMP uses 2D Delaunay Triangulation and now its code is open-sourced.

**Comment by lingkang** [comment]
感谢作者大大的开源。
编译过程中发现一些小的问题：
1. ACMP.cu line 562, 有一个typo， 多了一个分号。 
`const int center = p.y * cameras[0].width; + p.x;`
2. M_PI 貌似没有 define （不过这个可能是因为我在windows环境下编译的问题）

**Comment by GhiXu** [comment]
非常感谢您的反馈，已修正。目前代码都是在linux下测试的，在windows下可能需要做些调整。

**Comment by lingkang** [comment]
我做了以下修改之后已经编译成功：
1. 将 mkdir() 修改为Windows下对应的 _mkdir()函数
2. 添加 M_PI的 定义

因为之前稀疏重建部分输出格式的不同，目前还没有实际运行。
